lib/append_content_service.rb

```diff
@@ -102,6 +102,7 @@ def build_work(w_hsh)
   # If filename not given, use basename from path
   def build_file_set(path, filename=nil)
     fname = filename || File.basename(path)
+    puts "Processing: " + fname
```

Comment:
I recognize that these `puts` statements are following the convention in the file, but they should be converted to using a logger object. At the very least, the service should have a `Logger` that writes to STDOUT, so the binding isn't directly to `puts` and standard log levels can be used.

I will whip up a pull request that someone can test.